### ISSUE TITLE
Javascript exceptions thrown in background when navigating to any HTML based page in the nav tree

### ISSUE THREAD

**Issue by n-stein** [moneymanagerex/moneymanagerex#5434]
MMEX version:
 - [ ] 1.5.21
 - [ ] 1.6.0
 - [ ] 1.6.1
 - [x] 1.6.2 Alpha
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug
Upon navigating to any HTML-based page in the nav tree (Home, Budget Setup, Help, any report) a javascript exception is thrown in the background (visible in debug output).

Example when navigating "Home"
```
===== Model_Usage::AppendToUsage =================
{"module":"Home Page","seconds":0.294}


Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: UnifiedRegex::ParseError at memory location 0x0000006605CFB5F0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CF9090.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: UnifiedRegex::ParseError at memory location 0x0000006605CFB5F0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CF9090.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFB5D0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFBBF0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFBAF0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFB5D0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFB5C0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFC470.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: UnifiedRegex::ParseError at memory location 0x0000006605CFBA70.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CF9510.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFB5D0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFC020.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFBFB0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFB5E0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFBAF0.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFB710.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFBC30.
```

When navigating to "Help"
```
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFAF90.
Exception thrown at 0x00007FF9684DCD29 in mmex.exe: Microsoft C++ exception: Js::JavascriptExceptionObject at memory location 0x0000006605CFAE60.
```

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
Debug MMEX.
Open any DB.
Check debugger output.

**Comment by whalley** [comment]
Not sure if related to https://github.com/moneymanagerex/moneymanagerex/issues/5124 and the need to use a more modern web engine?

**Comment by n-stein** [comment]
I can confirm that using the Edge WebView2 backend for wxWebView resolves the issue, so this will be fixed by #5124.

**Comment by whalley** [comment]
Closing, fix will follow #5124
